src/functor.jl

```diff
@@ -109,6 +112,8 @@ adapt_storage(to::FluxCPUAdaptor, x::Zygote.FillArrays.AbstractFill) = x
 adapt_storage(to::FluxCPUAdaptor, x::T) where T <: CUDA.CUSPARSE.CUDA.CUSPARSE.AbstractCuSparseMatrix = adapt(Array, x)
 adapt_storage(to::FluxCPUAdaptor, x::Zygote.OneElement) = x
 adapt_storage(to::FluxCPUAdaptor, x::AbstractSparseArray) = x
+adapt_storage(to::FluxCPUAdaptor, x::CUDA.CURAND.RNG) = Random.default_rng()
```

Comment:
```suggestion
adapt_storage(to::FluxCPUAdaptor, x::CUDA.RNG) = Random.default_rng()
```
ref. https://github.com/JuliaGPU/CUDA.jl/blob/master/src/random.jl